codalab/lib/upload_manager.py

```diff
@@ -255,7 +255,7 @@ def write_fileobj(
             conn_str = os.environ.get('AZURE_STORAGE_CONNECTION_STRING', '')
             os.environ['AZURE_STORAGE_CONNECTION_STRING'] = bundle_conn_str
         try:
-            CHUNK_SIZE = 16 * 1024
+            CHUNK_SIZE = 1024 * 1024
```

Comment:
Why increase?

Reply:
The upload speed with the smaller chunk size was too slow due to the sleep behavior that occurs on the faster reader, which is always the index reader.

Reply:
It seems like there's no super meaningful reason to keep chunk size smallish since the speed tradeoff is too large